compiler/next/lib/queries/Context.cpp

```diff
@@ -461,6 +459,23 @@ bool Context::hasFilePathForId(ID id) {
   return false;
 }
 
+void Context::setFilePathForModuleID(ID moduleID, UniqueString path) {
+  UniqueString moduleIdSymbolPath = moduleID.symbolPath();
+  auto tupleOfArgs = std::make_tuple(moduleIdSymbolPath);
+
+  updateResultForQuery(filePathForModuleIdSymbolPathQuery,
+                       tupleOfArgs, path,
+                       "filePathForModuleIdSymbolPathQuery",
+                       /* isInputQuery */ false,
```

Comment:
Ignorance: If we have `QUERY_BEGIN_INPUT` and `QUERY_STORE_INPUT_RESULT`, can we not just have those record that the query is an input query so we don't have to pass it here?

Reply:
Yes I think we do that elsewhere. But this query is special and in fact updateResultForQuery isn't really even a regular part of the API (rather, QUERY_STORE_INPUT_RESULT is the API).

Reply:
(and all this PR did with this function is move it closer to the other code working with the same query).

Reply:
(filePathForModuleIdSymbolPathQuery is special because it is not an input query but it is set by the code assigning IDs from source files).